packages/lib/services/noteList/renderViewProps.ts

```diff
@@ -10,15 +10,16 @@ export interface RenderViewPropsOptions {
 	// only available on the `app-desktop` package. So the caller needs to pre-render the title and
 	// pass it as an option.
 	noteTitleHtml: string;
+	dateTimeFormat?: string;
 }
 
 const renderViewProp = (name: ListRendererDependency, value: any, options: RenderViewPropsOptions) => {
 	const renderers: Partial<Record<ListRendererDependency, ()=> string>> = {
-		'note.user_updated_time': () => time.unixMsToLocalDateTime(value),
-		'note.user_created_time': () => time.unixMsToLocalDateTime(value),
-		'note.updated_time': () => time.unixMsToLocalDateTime(value),
-		'note.created_time': () => time.unixMsToLocalDateTime(value),
-		'note.todo_completed': () => value ? time.unixMsToLocalDateTime(value) : '',
+		'note.user_updated_time': () => time.formatMsToLocal(value, options.dateTimeFormat),
+		'note.user_created_time': () => time.formatMsToLocal(value, options.dateTimeFormat),
+		'note.updated_time': () => time.formatMsToLocal(value, options.dateTimeFormat),
+		'note.created_time': () => time.formatMsToLocal(value, options.dateTimeFormat),
+		'note.todo_completed': () => value ? time.formatMsToLocal(value, options.dateTimeFormat) : '',
```

Comment:
If you look at `formatMsToLocal` you see that it's taking a default argument, right? And what is the value of the default argument. Isn't it already the user preferred date/time format?

Reply:
Yeah that makes sense. I have made the change and also made some changes to the test.